scripts/build_llvm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+# Args: LLVM_REPO LLVM_VERSION
+
+if [[ -z "${LLVM_HOME}" ]]; then
+    echo "Error! Environmental variable for LLVM_HOME is not set!"
+    exit 127
+fi
+
+LLVM_REPO=$1
+LLVM_VERSION=$2
+
+if [[ -z "${LLVM_REPO}" ]] || [[ -z "${LLVM_VERSION}" ]]; then
+    echo "Error! Parameters for LLVM repository are incorrect"
+    exit 127
+fi
+
+# Get the source code
+mkdir -p $LLVM_HOME
+cd $LLVM_HOME
+git clone $LLVM_REPO llvm_src
```

Comment:
What about naming folders `llvm-$LLVM_VERSION`, `build-$LLVM_VERSION`, `bin-$LLVM_VERSION`?

For the Docker use this will not matter, but for people running this on real machine this may matter, as they may already have llvm, build and bin folders.